### ISSUE TITLE
Odd rendering issues

### ISSUE THREAD

**Issue by Esseit** [Create-Additions/CreateAutomated#9]
There seems to be a rendering issue with the newest flywheel and create versions. all create pallet blocks dissapear when a cog is attatched to a automatic drill, and large create moving structures unfold and have massive graphical rendering errors. no crash or crash report though. Create version used is 0.3.2d and flywheel version used is 0.2.3. if i can provide any further information let me know and i can send what i have.

**Comment by Esseit** [comment]
nevermind, issue was resolved after removing mod but then showed itself again later. sorry about that

**Comment by kotakotik22** [comment]
can you explain? did the issue show again or the palette blocks?
> 
> 
> nevermind, issue was resolved after removing mod but then showed itself again later. sorry about that

**Comment by Esseit** [comment]
I figured it out. the error in the create pallete blocks rendering was caused by the render regions in optifine. it seemed to get finicky when blocks were placed on very specific cords. The unfolding textures issue was caused by the map mod Voxel Map, when rendering too many create structures the mini map freaks out. its hard to explain how, but i suggest testing it yourself if youre curious. 
 I tested this using a circle of drills on a rope pulley that was 25 blocks in diameter.

> can you explain? did the issue show again or the palette blocks?
